### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by BJenning**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
   The GDX Analytics Drupal Snowplow module installs and runs Snowplow 
   Javascript web trackers, and provides an interface to configure them.
   
+## Requirements  
+
+  This module requires Drupal 8.
+  
 ## Project Status
 
   This project is currently under development.
```
Comment (ryrjanes) [inline README.md]: needs 'and actively supported by the GDX Analytics Team.' added

File: README.md [hunk 2]
```diff
@@ -3,6 +3,10 @@
   The GDX Analytics Drupal Snowplow module installs and runs Snowplow
```
Comment (ryrjanes) [inline README.md:3]: needs to change to 'This is the central repository for work by the GDX Analytics Team.'

File: README.md [hunk 3]
```diff
@@ -32,9 +36,9 @@
...
   the uri of the tracking script.
 
-## Getting Help
-
-For any questions regarding this project, please contact the GDX Analytics Team.
+## Getting Help or Reporting an Issue
+ 
+For any questions regarding this project, or for inquiries about starting a new analytics account, please contact the GDX Analytics Team.
 
 ## Contributors
```
Comment (ryrjanes) [inline README.md:44]: needs to change to 'The GDX Analytics Team are the main contributors to this project and maintain the code.'

**Review by ryrjanes — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: README.md (hunk 1 above)
Comment (BJenning) [reply README.md]: Added the correct wording.

File: README.md (hunk 2 above)
Comment (BJenning) [reply README.md:3]: Done.

File: README.md (hunk 3 above)
Comment (BJenning) [reply README.md:44]: We arent changing the contributing instructions yet.

Comment (BJenning) [reply README.md:44]: Also, i got the getting help section directly from the confluence doc we are working from.

Comment (BJenning) [reply README.md:44]: I saw Dan's comment in the chat, so I updated the wording.

**Review by ryrjanes — APPROVED** [review]
looks good with changes.

**Pull request merged**